src/entities/mentor/ui/mentor-feedback-card/mentor-feedback-card.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import { useState } from 'react';
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+import { MentorFeedback } from '../../types';
+import { Modal } from '@/shared/ui/modal';
+
+import styles from './mentor-feedback-card.module.scss';
+
+const cx = classNames.bind(styles);
+
+type MentorFeedbackCardProps = MentorFeedback;
+
+const FEEDBACK_MAX_CHARS = 500;
+
+export const MentorFeedbackCard = ({ name, course, review, photo }: MentorFeedbackCardProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const handleOpenModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleCloseModal = () => {
+    setIsModalOpen(false);
+  };
+
+  const isLongReview = review.length > FEEDBACK_MAX_CHARS;
+
+  const card = (
+    <div className={cx('card')}>
+      <div className={cx('card-info')} data-testid="card-info">
+        <Image
+          className={cx('card-picture')}
+          src={photo}
+          alt={`${name} ${course}`}
+          data-testid="mentor-photo"
+        />
+        <header className={cx('card-header')}>
+          <h3 className={cx('card-title')} data-testid="card-title">
+            {name}
+          </h3>
+          <h4 className={cx('card-subtitle')} data-testid="card-subtitle">
+            <b>Course: </b>
+            {course}
+          </h4>
+        </header>
+      </div>
+      <div className={cx('card-content-wrapper')} data-testid="card-content-wrapper">
+        <p className={cx('card-content')} data-testid="card-content">
```

Comment:
Can we use our `Paragraph` component here?

Reply:
I don't see much sense in this, as it would require overriding a lot of styles